helm/aphw-ddi-enforcement/templates/config-map.yaml

```diff
@@ -6,8 +6,10 @@ data:
   APPINSIGHTS_CLOUDROLE: {{ quote .Values.name }}
   {{- if .Values.pr }}
   REDIRECT_URL: https://{{ .Values.ingress.endpoint }}-{{ .Values.pr }}.{{ .Values.ingress.server }}/{{ .Values.azureAppRegistration.redirectUrl }}
+  POST_LOGOUT_URL: https://{{ .Values.ingress.endpoint }}-{{ .Values.pr }}.{{ .Values.ingress.server }}/{{ quote .Values.container.postLogoutRoute }}
```

Comment:
I'm not 100% sure these ingress values will be correct, especially when we have external access, but will approve as is, and we can enhance this later